README.md

```diff
@@ -355,8 +384,16 @@ function App() {
 
 ## Contributing
 
-Before submitting a pull request, please read our [Contributing Guide](.github/CONTRIBUTING.md). It contains a set of instructions and guidelines that aim to make the contribution process easy and frictionless.
+Thanks for being interested on making this package better. We encourage everyone to help improving this project with some new features, bug fixes and performance issues. Please take a little bit of your time to read our guides, so this process can be faster and easier.
+
+### Contributing Guide
```

Comment:
`Contributing Guide` -> `Contribution Guide`;

Or perhaps stick to the previous pattern `Contribution Guidelines`, or change all to `Contribution Guide`?

What do you guys think? Which one should we keep?

Reply:
Make it `Contribution Guidelines` so it stays consistent with the `CONTRIBUTING.md` title, btw, I gotta change a similar typo there.